Libraries/Components/Pressable/useAndroidRippleForView.js

```diff
@@ -27,6 +27,7 @@ export type RippleConfig = {|
   color?: ColorValue,
   borderless?: boolean,
   radius?: number,
+  useForeground?: boolean,
```

Comment:
Can you just name this `foreground`? Even though it is a property name and not a variable name, avoiding the "use" prefix is just better in the current world with hooks.